vernac.h

```diff
@@ -37,6 +37,11 @@ command *make_data(variable* name, int num_constructors, int num_params);
 command *make_axiom(variable* name, term* ty);
 command *make_import(variable* name);
 
+// A helper that makes dealing with input streams easier from Rust.
```

Comment:
Rust has a cross-platform (windoze, osx, linux) abstraction for files and input streams, this seemed like the least crazy thing I could do.